src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -284,6 +284,11 @@ protected int doWork() {
         log.info("Loading up the target reference genome.");
         final ReferenceSequenceFileWalker walker = new ReferenceSequenceFileWalker(REFERENCE_SEQUENCE);
         final Map<String, ReferenceSequence> refSeqs = new HashMap<>();
+        // check if sequence dictionary exists
+        if (walker.getSequenceDictionary() == null) {
+            log.error("Reference " + REFERENCE_SEQUENCE.getAbsolutePath() + " must have associated Dictionary .dict file in same directory");
```

Comment:
May as well write this using a complete sentence, e.g. `"Reference x must have an associated Dictionary .dict file in the same directory.`